sdk/analytics/src/main/java/com/klaviyo/analytics/state/KlaviyoState.kt

```diff
@@ -128,7 +128,6 @@ internal class KlaviyoState : State {
         _phoneNumber.reset()
         _anonymousId.reset()
         _attributes.reset()
-        _pushToken.reset()
```

Comment:
Yeah. Since there is no near term need for a full reset I'm just vary of leaving dead code paths in. However, you have better context of this codebase so let me know if you think otherwise.